### PR TITLE
[Android] Add 'Bug report' menu item

### PR DESCRIPTION
**Pull request by adamradocz**
#2937 
This PR does not close the issue, because this feature can be implemented in the desktop version too.

**Description of the Change**
- Add 'Bug report' menu item, which redirects to [GitHub/issues](https://github.com/BOINC/boinc/issues). 
- 'Help menu' link changed to https.
- Switch Bug report and Event Log menu icons.

**Release Notes**
[Android] Add Bug report menu item.

**Screenshot**
screenshot_20181227-170424_boinc

### CODE REVIEW

**Comment by AenBleidd** [comment]
Could anyone please review this?

**Comment by JuhaSointusalo** [comment]
Commented in https://github.com/BOINC/boinc/pull/2941#issuecomment-451576177 .

File: android/BOINC/app/src/main/java/edu/berkeley/boinc/BOINCActivity.java [hunk 1]
```diff
@@ -271,8 +271,10 @@ private void dispatchNavBarOnClick(NavDrawerItem item, boolean init) {
 				fragmentChanges = true;
 				break;
 	    	case R.string.menu_help:
-	    		Intent i = new Intent(Intent.ACTION_VIEW, Uri.parse("http://boinc.berkeley.edu/wiki/BOINC_Help"));
-	    		startActivity(i);
+	    		startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://boinc.berkeley.edu/wiki/BOINC_Help")));
+	    		break;
+	    	case R.string.menu_bug_report:
+	    		startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://boinc.berkeley.edu/forum_forum.php?id=24")));
```
Comment (brevilo) [inline android/BOINC/app/src/main/java/edu/berkeley/boinc/BOINCActivity.java]: Please use the same link as for the Manager, which just got [decided](https://github.com/BOINC/boinc/pull/2941#issuecomment-452515127) upon.

File: android/BOINC/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -366,6 +366,7 @@
     <string name="menu_run_mode_enable">Resume</string>
     <string name="menu_about">About</string>
     <string name="menu_help">Help</string>
+    <string name="menu_bug_report">Bug report</string>
```
Comment (brevilo) [inline android/BOINC/app/src/main/res/values/strings.xml]: Please use 'Report Issue' to match the manager.

File: android/BOINC/app/src/main/java/edu/berkeley/boinc/BOINCActivity.java (hunk 1 above)
Comment (adamradocz) [reply android/BOINC/app/src/main/java/edu/berkeley/boinc/BOINCActivity.java]: Done.

Comment (adamradocz) [reply android/BOINC/app/src/main/java/edu/berkeley/boinc/BOINCActivity.java]: Done.

File: android/BOINC/app/src/main/res/values/strings.xml (hunk 2 above)
Comment (adamradocz) [reply android/BOINC/app/src/main/res/values/strings.xml]: Done.

**Pull request merged**